skimage/measure/_regionprops.py

```diff
@@ -901,6 +1039,71 @@ def perimeter(image, neighbourhood=4):
     return total_perimeter
 
 
+def crofton_perimeter(image, directions=4):
+    """Calculate total perimeter of all objects in binary image, based on
+    Crofton formula. Be aware that there is no unique perimeter function, which 
```

Comment:
I'd rather say. As `measure.perimeter`, this function returns an approximation of the perimeter in continuous space.

Reply:
the "be aware" can be intimidating for users :-)

Reply:
ok